extensions/sql-database-projects/package.json

```diff
@@ -377,20 +398,34 @@
         {
           "command": "sqlDatabaseProjects.createProjectFromDatabase",
           "when": "nodeType =~ /^(Database|Server)$/ && connectionProvider == MSSQL && mssql:engineedition != 11",
-          "group": "export"
+          "group": "export@1"
+        },
+        {
+          "command": "sqlDatabaseProjects.updateProjectFromDatabase",
+          "when": "nodeType =~ /^(Database|Server)$/ && connectionProvider == MSSQL && mssql:engineedition != 11",
+          "group": "export@2"
         }
       ],
       "dataExplorer/context": [
         {
           "command": "sqlDatabaseProjects.createProjectFromDatabase",
           "when": "nodeType =~ /^(Database|Server)$/ && connectionProvider == MSSQL && mssql:engineedition != 11",
-          "group": "export"
+          "group": "export@1"
+        },
+        {
+          "command": "sqlDatabaseProjects.updateProjectFromDatabase",
+          "when": "nodeType =~ /^(Database|Server)$/ && connectionProvider == MSSQL && mssql:engineedition != 11",
+          "group": "export@2"
         }
       ],
       "dashboard/toolbar": [
         {
           "command": "sqlDatabaseProjects.createProjectFromDatabase",
           "when": "connectionProvider == 'MSSQL' && mssql:engineedition != 11"
+        },
+        {
+          "command": "sqlDatabaseProjects.updateProjectFromDatabase",
```

Comment:
I'm a little concerned that the dashboard toolbar will start getting too cluttered with too many commands and they'll start to get hidden in the overflow menu and not be quickly accessible anymore. It would be good to discuss with designers on what principles we should have on what should be added to toolbars, like if they should be frequently used commands or if there should be a limit on the number of contributed toolbar commands per each extension.